### DISCUSSION TITLE
NPT in ice_in

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#808] [Q&A]
Similar to the questions about ww3_multi.inp, could someone point me to the calculation for NPT in ice_in? Thanks!

**Comment by DeniseWorthen** [comment]
npt is not used by the coupled model. When running standalone, it controls the number of time steps in units of npt_units

**Reply by benjamin-cash** [reply]
Thanks @DeniseWorthen. I was asking because NPT is different in the ice_in files for the 2011070100 and 2013040100 p7 cases I have been looking at, but it sounds like I can safely ignore that?

**Comment by DeniseWorthen** [comment]
I would suggest keeping track of the ice_in_template in the RT system. If it is not set as a variable sustitution (@[variableName]) then it is not something to worry about. 

I don't know what or how the workflow manages the ice_in so if your issue is w/ the workflow, I'd ask there.

**Reply by benjamin-cash** [reply]
So far as I know there isn't an RT for the P7 configuration, so I am having to reverse engineer the settings from the P7 run directories I have access to. I have run directories for two different cases and am working my way through things like ice_in with diff to see what was changed, which is how I found NPT is different for the two runs.

**Reply by JessicaMeixner-NOAA** [reply]
@benjamin-cash you have the whole workflow that's used for P7, it's here: https://github.com/noaa-emc/global-workflow/tree/feature/coupled-crow

**Reply by DeniseWorthen** [reply]
I still suggest keeping track of which settings in ice_in are actually potentially changeable.

**Reply by benjamin-cash** [reply]
Unless there has been a development I don't know about the workflow doesn't run on the XSEDE systems. It hasn't been ported to Expanse at all, and Walter tried for weeks to get it to work on Stampede2 without success. So using this approach with my own cylc workflow is the only path forward that I have been able to come up with for those systems.
